libs/feature/search/src/lib/fuzzy-search/fuzzy-search.component.ts

```diff
@@ -8,7 +7,10 @@ import {
   ViewChild,
 } from '@angular/core'
 import { SearchApiService } from '@geonetwork-ui/data-access/gn4'
-import { AutocompleteComponent } from '@geonetwork-ui/ui/inputs'
+import {
+  AutcompleteItem,
```

Comment:
Just had a quick look over the big PR, seeing one typo here.

Reply:
Good catch, it was introduced in a previous PR, I will fix it thanks